docs/attributes-registry/faas.md

```diff
@@ -0,0 +1,93 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: FaaS
+--->
+
+# FaaS
+
+## FaaS Attributes
+
+<!-- semconv registry.faas(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `faas.coldstart` | boolean | A boolean that is true if the serverless function is executed for the first time (aka cold-start). |  |
+| `faas.cron` | string | A string containing the schedule period as [Cron Expression](https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm). | `0/5 * * * ? *` |
+| `faas.document.collection` | string | The name of the source on which the triggering operation was performed. For example, in Cloud Storage or S3 corresponds to the bucket name, and in Cosmos DB to the database name. | `myBucketName`; `myDbName` |
+| `faas.document.name` | string | The document name/table subjected to the operation. For example, in Cloud Storage or S3 is the name of the file, and in Cosmos DB the table name. | `myFile.txt`; `myTableName` |
+| `faas.document.operation` | string | Describes the type of the operation that was performed on the data. | `insert` |
+| `faas.document.time` | string | A string containing the time when the data was accessed in the [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) format expressed in [UTC](https://www.w3.org/TR/NOTE-datetime). | `2020-01-23T13:47:06Z` |
+| `faas.instance` | string | The execution environment ID as a string, that will be potentially reused for other invocations to the same function/function version. [1] | `2021/06/28/[$LATEST]2f399eb14537447da05ab2a2e39309de` |
+| `faas.invocation_id` | string | The invocation ID of the current function invocation. | `af9d5aa4-a685-4c5f-a22b-444f80b3cc28` |
+| `faas.invoked_name` | string | The name of the invoked function. [2] | `my-function` |
+| `faas.invoked_provider` | string | The cloud provider of the invoked function. [3] | `alibaba_cloud` |
+| `faas.invoked_region` | string | The cloud region of the invoked function. [4] | `eu-central-1` |
+| `faas.max_memory` | int | The amount of memory available to the serverless function converted to Bytes. [5] | `134217728` |
+| `faas.name` | string | The name of the single function that this runtime instance executes. [6] | `my-function`; `myazurefunctionapp/some-function-name` |
+| `faas.time` | string | A string containing the function invocation time in the [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) format expressed in [UTC](https://www.w3.org/TR/NOTE-datetime). | `2020-01-23T13:47:06Z` |
+| `faas.trigger` | string | Type of the trigger which caused this function invocation. | `datasource` |
+| `faas.version` | string | The immutable version of the function being executed. [7] | `26`; `pinkfroid-00002` |
+
+**[1]:** * **AWS Lambda:** Use the (full) log stream name.
+
+**[2]:** SHOULD be equal to the `faas.name` resource attribute of the invoked function.
```

Comment:
Not a part of current refactoring but for maybe future PR
I see here a duplication of data for `invoked_name`, `invoked_provider` and `invoked_region`. Was it needed because of strict separation between resource and not resource attributes? Should we remove it now and replace with original fields as we are going into removing this restriction in build tools?

Reply:
@trisch-me could you please open an issue to track your concerns here? Then we can merge this and not forget to follow up later.

Reply:
Created #799